app/v3/notifications/rest.py

```diff
@@ -0,0 +1,29 @@
+""" Implement v3 endpoints for the Notification model. """
+
+from app.models import EMAIL_TYPE, SMS_TYPE
+from app.v3.notifications.notification_schemas import notification_v3_post_request_schema
+from flask import Blueprint, request
+from jsonschema.validators import Draft202012Validator
+from uuid import uuid4
+
+v3_notifications_blueprint = Blueprint("v3_notifications", __name__, url_prefix='/notifications')
+
+# Create an instance of Draft202012Validator to validate post request data.  Unit tests
+# should call jsonschema.validate, which also verifies that the schema is valid.
+v3_notifications_post_request_validator = Draft202012Validator(notification_v3_post_request_schema)
```

Comment:
This is outside the route handler so it doesn't get recreated with every request.  The goal is to minimize response time.